contracts/MarginCalculator.sol

```diff
@@ -355,8 +355,8 @@ contract MarginCalculator is Initializable {
      * @dev convert uint256 to FixedPointInt, no scaling invloved
      * @return the FixedPointInt format of input
      */
-    function _uint256ToFixedPointInt(uint256 _num) internal pure returns (FixedPointInt256.FixedPointInt memory) {
-        return FixedPointInt256.FixedPointInt(SignedConverter.uintToInt(_num));
+    function _uint256ToFPI(uint256 _num) internal pure returns (FPI.FixedPointInt memory) {
```

Comment:
This function is not doing any scaling, right ?

Reply:
no, I only rename it to a shorter name